packages/web-components-config-server/src/routes/web-components-config/api.ts

```diff
@@ -31,13 +39,26 @@ export const createConfig = async ({ traceId, data }: CreateParams): Promise<Web
   }
 }
 
-export const updateConfig = async ({ traceId, data }: UpdateParams): Promise<WebComponentConfig> => {
+export const patchConfig = async ({ traceId, data }: UpdateParams): Promise<WebComponentConfig> => {
   try {
-    logger.info('Updating config...', { traceId, data })
-    const { customerId, ...rest } = data
-    const oldItem = await getConfigByClientId({ traceId, data: { customerId } })
+    logger.info('Patching config...', { traceId, data })
+    const { customerId, appId, ...rest } = data
+    const oldItem = await getConfigByClientId({ traceId, data: { customerId, appId } })
     const itemToUpdate = generateSchemaItem({ ...oldItem, ...rest })
     const result = await dynamoDBMapper.update(itemToUpdate)
+    logger.info('Patch config successfully', { traceId, result })
+    return result
+  } catch (error) {
+    await logger.error('Patch config failed', { traceId, error: stringifyError(error) })
+    throw error
+  }
+}
+
+export const putConfig = async ({ traceId, data }): Promise<WebComponentConfig> => {
+  try {
+    logger.info('Updating config...', { traceId, data })
+    const itemToUpdate = generateSchemaItem(data)
+    const result = await dynamoDBMapper.put(itemToUpdate)
```

Comment:
just wonder why we need both PUT, and PATCH to update ?

Reply:
@phmngocnghia please fix to the next PR